gnosis-safe-deploy/scripts/executeTransaction.js

```diff
@@ -0,0 +1,86 @@
+const EthersAdapter = require('@gnosis.pm/safe-ethers-lib').default;
+const { default: Safe, EthSignSignature } = require('@gnosis.pm/safe-core-sdk');
+const SafeServiceClient = require('@gnosis.pm/safe-service-client').default;
+const fortaAgent = require('forta-agent');
+
+const { ethers } = fortaAgent;
```

Comment:
Same question about using `ethers` from `forta-agent` instead of the `ethers` package directly

Reply:
Reuse from proposal script.

Fixed.